openwisp_monitoring/monitoring/migrations/0013_merge_traffic_metrics.py

```diff
@@ -0,0 +1,23 @@
+from django.db import migrations
+
+
+def merge_traffic_metrics(apps, schema_editor):
+    Metric = apps.get_model('monitoring', 'Metric')
+    rx_metrics = Metric.objects.filter(field_name='rx_bytes')
+    for metric in rx_metrics:
+        metric.name = f'{metric.name.split()[0]} traffic'
+        metric.save()
```

Comment:
I think it would be useful to provide feedback to the user:

```
new_name = f'{metric.name.split()[0]} traffic'
print('Renamed metric "{}" to "{}"'.format(metric.name, new_name))
metric.name = new_name
```